sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/KnownLinuxVirtualMachineImage.java

```diff
@@ -8,18 +8,36 @@
     UBUNTU_SERVER_16_04_LTS("Canonical", "UbuntuServer", "16.04.0-LTS"),
     /** UbuntuServer 18.04LTS. */
     UBUNTU_SERVER_18_04_LTS("Canonical", "UbuntuServer", "18.04-LTS"),
+
     /** Debian 9. */
     DEBIAN_9("credativ", "Debian", "9"),
     /** Debian 10. */
     DEBIAN_10("Debian", "debian-10", "10"),
+
     /** CentOS 8.1. */
     CENTOS_8_1("OpenLogic", "CentOS", "8_1"),
-    /** OpenSUSE-Leap 15.1. */
-    OPENSUSE_LEAP_15_1("SUSE", "openSUSE-Leap", "15-1"),
-    /** SLES 15-SP1-gen1. */
-    SLES_15_SP1("SUSE", "sles-15-sp1", "gen1"),
```

Comment:
@ChenTanyi 

15.1 is gone.

I figure I will just use e.g. `SLES_15` instead of `15.x`.

Reply:
OK